### DISCUSSION TITLE
meta-tags and i18n

### DISCUSSION THREAD

**Discussion by Jindorf** [vikejs/vike#1098] [Discussion]
Hey,

how do you approach meta tags with i18n in VPSv1?
I've set up title and description as suggested in the [docs](https://vite-plugin-ssr.com/meta#create-new-configurations), but i need the document.title to be i18n-ready. I use Vue3 and Vue-i18n for i18n.

Can I approach this directly with VpsV1? Or should i just use vueuse/head ?

**Comment by brillout** [comment]
One thing you could do is pass a function instead of string, that's what [vike-vue](https://github.com/vikejs/vike-vue) does.

```js
// /pages/product/+title.js

export default pageContext => `${pageContext.lang == 'fr' ? 'Produit' : 'Product'} ${pageContext.product.name}`
```

If you don't need it to be dynamic you can simplify it:

```js
// /pages/about/+title.js

export default {
  en: 'About us',
  de: 'Über uns'
}
```
While updating your `renderer/` accordingly.
